### ISSUE TITLE
The server crashes on specific navigation paths

### ISSUE THREAD

**Issue by garro95** [rinigus/osmscout-server#435]
Scout Server version 3.0.0 on Ubuntu Touch systematically crashes when asking navigation paths with PureMaps 3.2.0.
Also, in the area, any point on the map is shown as "No name" and search does not work, even though the map seems to contain all the information.

Downloaded maps are North America / US / Regions / US Midwest and North America / US / Regions / West
I also tried with and without the North America / US / Pennsylvania map

Session log
```
cat .cache/osmscout-server/osmscout-server/session.log 
INFO:18:25:11 Request: /v1/activate
INFO:18:25:11 Request: /v1/mbgl/style?style=osmbright
INFO:18:25:18 Request: /v1/activate
INFO:18:25:18 Request: /v1/mbgl/tile?x=17&y=24&z=6
INFO:18:25:18 Request: /v1/mbgl/tile?x=284&y=386&z=10
INFO:18:25:18 Request: /v1/mbgl/tile?x=284&y=385&z=10
INFO:18:25:18 Request: /v1/mbgl/sprite@2x.json
INFO:18:25:18 Request: /v1/mbgl/sprite@2x.png
INFO:18:25:18 Request: /v1/mbgl/glyphs?stack=Noto Sans Bold&range=0-255
INFO:18:25:18 Request: /v1/mbgl/glyphs?stack=Noto Sans Regular&range=0-255
INFO:18:25:18 Request: /v2/route?json={"locations": [{"lat": 40.437570149387845, "lon": -79.99890298140497, "name": "Punto senza nome", "type": "break_through"}, {"lat": 40.431043424335996, "lon": -79.96043575728753, "name": "Punto senza nome", "type": "break"}], "costing": "auto", "costing_options": {"auto": {"use_ferry": 0.5, "use_highways": 1, "use_tolls": 0.5}}, "directions_options": {"language": "it", "units": "kilometers"}}
INFO:18:25:18 Request: /v1/mbgl/glyphs?stack=Noto Sans Italic&range=0-255
INFO:18:25:18 Request: /v1/mbgl/glyphs?stack=Open Sans Regular&range=0-255
```

**Comment by rinigus** [comment]
Usually, such crashes are caused by 

- some damage in map files
- QML cache getting corrupted

While QML cache issue usually is related to updates, try to remove it first as it is easier. No idea where it is hiding in Ubuntu, though.

Then try to move all maps into some other folder and download again all the maps. Please try to find the smallest set of maps that leads to this bug. It is not normal that you have a region downloaded and it is not showing up in search and POI recognition is not working.

Let me know how it went.

**Comment by jonnius** [comment]
Here is where to find

- QML cache: `/home/phablet/.cache/pure-maps.jonnius/pure-maps.jonnius/qmlcache/`
- Maps: `/home/phablet/Maps.OSM/`

**Comment by rinigus** [comment]
I will make a new import - which has been due anyway. Something is badly off in this one. But it would take some time, probably a week or two

**Comment by rinigus** [comment]
Import done - same issue. Looks like my partitioning between regions is wrong. Will look into it and distribute new maps when that is ready

**Comment by rinigus** [comment]
So, issue was in splitting map data into regions. It has been fixed now and the main impact was for the following parts:

- asia/azerbaijan
- europe/germany/bayern/unterfranken
- north-america/us/kentucky
- north-america/us/pennsylvania

New maps are being uploaded and I hope they will be ready on the server soon. "Soon" could take couple of days...

Closing it here. Please reopen if it still a problem with the new maps.

Fix in commit: 56421023ff70b345f8a82654b13801aa40ac6da4

**Comment by rinigus** [comment]
New maps are out.
